www/src/models/feed.ts

```diff
@@ -0,0 +1,55 @@
+export class EditorModel {
+  static code: string = '';
+  static ready: boolean = false;
+  static language: string = 'python3';
+  static editorLanguage: string = 'python';
+  static monacoModel: monaco.editor.ITextModel;
+
+  static setCode(code: string) {
+    EditorModel.code = code;
+  };
+
+  static getCode(): string {
+    return EditorModel.code;
+  }
+
+  static setEditorLanguage(language: string) {
```

Comment:
I don't see directly why would need monaco here. BUT i know better and know that you have your reasons. All i want is a explanation so i can understand better. 👍

Reply:
I hade to refresh twice while reviewing. I couldn't remove this comment after second refresh though. 💃

Reply:
Hehe, yes, there's no reason for that file, just copied it from editor model as a boilerplate and forgot to remove it.